quiz/backend/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from django.core.management import call_command
+
+
+@pytest.fixture
+def generated_data_with_custom_command():
+    """Fixture for calling create_test_data command
+
+    :return func call_command: call_command
+    """
+    return call_command(
+        "create_test_data",
+        "5",
+    )
```

Comment:
brakuje entera

Reply:
minus mi się nie wyświetla, patrzyłem tylko na pliki pythona, że nie ma  pustej linii na końcu pliku, ale to może u mnie jakieś błędy w konfiguracji, Przepraszam Cię za niepotrzebne zamieszanie. 
Generalnie super robota! Duży Aprove